ag/enrollment.go

```diff
@@ -5,6 +5,12 @@ import (
 	"time"
 )
 
+//gracePeriod is the grace-Period for deliveries after the deadline. It can only be <int> hours.
+//Currently there is a 2 hour grace period. It should be between [0,1,..,23]
+const gracePeriod time.Duration = time.Duration(2 * time.Hour)
+
+//Keep in mind this will be the same across all enrollments (courses) on the AG service.
```

Comment:
Please move this comment above the definition of the grace period, and rephrase, e.g. "Note that the grace period applies to all enrollments (courses)."